skyrim-platform/src/platform_se/skyrim_platform/TextApi.cpp

```diff
@@ -1,18 +1,21 @@
 #include "TextApi.h"
-#include <fstream>
+
 #include <string>
 
 #include <codecvt>
 #include <locale>
 
 namespace TextApi {
+
 JsValue TextApi::CreateText(const JsFunctionArguments& args)
 {
   std::array<double, 4> argColor;
 
   auto argPosX = static_cast<double>(args[1]);
   auto argPosY = static_cast<double>(args[2]);
-  auto argString = static_cast<std::string>(args[3]);
+
+  std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
+  auto argString = converter.from_bytes(static_cast<std::string>(args[3]));
```

Comment:
Actually, this conversion method was marked as deprecated in C++17. But sadly there's no other usable alternative in standard C++ library (though technically I could use WinAPI for that, but it's a C function, not C++, and could be more portable). Since it's not going away in C++20 or C++23, I think it's safe to use for now.

Sadly, I couldn't just write `auto argString = static_cast<std::wstring>(args[3]);`: the implementation in either our `JsEngine` or Chakra itself is broken and returns empty string.

See #981